tracee/tracee.go

```diff
@@ -141,7 +147,14 @@ func New(cfg TraceeConfig) (*Tracee, error) {
 	if cfg.CaptureFilesWrite {
 		essentialEvents[EventsNameToID["vfs_write"]] = false
 	}
-
+	if cfg.SecurityAlerts || cfg.ExtractDynamicCode {
+		essentialEvents[EventsNameToID["mmap"]] = false
+		essentialEvents[EventsNameToID["mprotect"]] = false
+	}
+	if cfg.ExtractDynamicCode {
```

Comment:
don't we need the *_alert events if SecurityAlerts is chosen?

Reply:
This is done in the main.go already